charts/consul/test/terraform/eks/main.tf

```diff
@@ -53,8 +53,8 @@ module "vpc" {
 module "eks" {
   count = var.cluster_count
 
-  source  = "terraform-aws-modules/eks/aws"
-  version = "17.24.0"
+  source                 = "terraform-aws-modules/eks/aws"
+  version                = "17.24.0"
```

Comment:
did you want these here?

Reply:
It automatically fmt'd but I'll take this out for now